server/controller/accountController.js

```diff
@@ -136,7 +136,10 @@ class AccountController {
   static async getAllBankAccounts(req, res) {
     let accounts;
     try {
-      accounts = await pool.query('SELECT email, accounts.* FROM users JOIN accounts on users.id = accounts.OWNER');
+      const { status } = req.query;
+      if (status === 'active') {
+        accounts = await pool.query('SELECT email, accounts.* FROM users JOIN accounts on users.id = accounts.OWNER where status = \'active\'');
```

Comment:
This line and the line after are too long, next time try to break it into smaller lines. But you can merge for now

Reply:
Okay....I will do that...thank you for the feedback